R/tm_a_regression.R

```diff
@@ -241,24 +249,66 @@ srv_a_regression <- function(id,
   with_filter <- !missing(filter_panel_api) && inherits(filter_panel_api, "FilterPanelAPI")
   checkmate::assert_class(data, "tdata")
   moduleServer(id, function(input, output, session) {
-    anl_merged_input <- teal.transform::merge_expression_module(
+
+    rule_rvr1 <- function(value) {
+      if (isTRUE(input$plot_type == "Response vs Regressor")) {
+        if (length(value) > 1L)
+          "This plot can only have one regressor."
```

Comment:
So I think highlighting this ui is misleading:
![image](https://user-images.githubusercontent.com/15201933/209345758-3c3e3a50-d0bf-4015-9bda-da4564a3edb9.png)

as it is a valid regression -> if anything we should highlight the radiobutton

Reply:
similarly this one:
![image](https://user-images.githubusercontent.com/15201933/209347460-447ee804-c05a-479c-ac2c-60fecc4c9d99.png)

although unlike the case above, in this case I do wonder why we allow this for other plot types...

Reply:
> So I think highlighting this ui is misleading:
> (...)
> as it is a valid regression -> if anything we should highlight the radiobutton

This is the original message:
"Response vs Regressor plot is provided only for regressions with exactly one regressor.\n",
            "Choose another plot or reduce number of regressors"

If anything, we should highlight both.

I don't think printing all of that 👆 in input widgets is a good idea.